[Addon ]Update addon contribution docs. (#741)
oam-dev/kubevela.io: docs/platform-engineers/addon/intro.md

## Known Limits

> - Install the addon in the sub-cluster. If the addon is not installed in the control plane, but only in sub-clusters, there are known some issues.

Reviewer: 还有哪些问题，这里只说了有问题，没说是什么，吓得别人都不敢用了

## Extend Terraform Addon

> * We have build some tool for more convenience to extend cloud resource addons, you can refer to the [extend terraform addon docs](./terraform).

Reviewer: ```suggestion
* We have built some tools to extend cloud resource as addons  for convenience, you can refer to the [extend terraform addon docs](./terraform).
```

## Contribution

> If you make and use an addon in your product environment, welcome to contribute your customize addon to community. By doing this, whole community can help to test, optimize , upgrade or fix bugs for the addon. Community also can help you find more usage scenario about thr addon. 

Reviewer: ```suggestion
If you have developed your own addons, welcome to contribute to the [community catalog](https://github.com/kubevela/catalog). 
```

Reviewer: 这种开源的好处没必要在这里讲，大多数公司也不让。因为相比于有社区维护（这种非常潜在的可能性），你这里提到的暴露风险本身就是不好的

> Meanwhile, any bug fix about existing addon is also welcome. Contribution is making a pull request to [this](https://github.com/kubevela/catalog) repo. Please refer to [doc](https://github.com/kubevela/catalog/blob/master/README.md#contribution) for more info about contribution rules.

Reviewer: ```suggestion
Meanwhile, any bugfix of existing addons are welcomed. Just make a pull request to [this](https://github.com/kubevela/catalog) repo. 
```

鼓励别人贡献不用讲规则，规则是别人已经开始贡献了（提PR）发给人家的（合并PR的）门槛。
